server/package.json

```diff
@@ -4,10 +4,12 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
+    "bcrypt": "^3.0.2",
```

Comment:
Also see the `bcryptjs` package. FYI

Reply:
Yup good one, saw that other teams used it